grove/test_utilities.py

```diff
@@ -0,0 +1,92 @@
+"""Module containing utilities that will eventually be implemented in pyquil."""
```

Comment:
please move this file to a dedicated `tests` folder

Reply:
:+1: